### ISSUE TITLE
Legend Missing from Fieldset for Number Per Page on Dashboard Works

### ISSUE THREAD

**Issue by mtribone** [samvera-labs/nurax-pre2023#128]
### Descriptive summary

The dropdown to set the number of results per page on the works dashboard has a fieldset, but no legend for the fieldset. If a higher description of the functionality of the controls is not necessary, then the fiedset should be removed. 

### Rationale

A fieldset legend presents a description of the form elements within a fieldset and is especially useful to screen reader users. A legend should be provided when a higher level description is necessary for groups of check boxes, radio buttons, or other form controls.

### Expected behavior

Expect all fieldsets to have legends for better accessibility. 

### Actual behavior

Fieldset doesn not have a legend.

### Steps to reproduce the behavior

1. Navigate to your user dashboard
1. Select works from the left-hand side
1. Inspect the HTML associated with the number per page form controls

**Comment by mjgiarlo** [comment]
@mtribone Would you mind suggesting some text for the legend?

**Comment by mtribone** [comment]
In ScholarSphere we use `<legend class="sr-only">Number of results to display per page</legend>`
